python_modules/libraries/dagster-snowflake-pandas/dagster_snowflake_pandas_tests/test_snowflake_pandas_type_handler.py

```diff
@@ -128,6 +154,28 @@ def read_pandas_df(df: pandas.DataFrame):
             assert set(df.columns) == {"foo", "quux"}
             assert len(df.index) == 2
 
+        time_df = pandas.DataFrame(
+            {
+                "foo": ["bar", "baz"],
+                "date": [pandas.Timestamp("2017-01-01T12"), pandas.Timestamp("2017-02-01T12")],
```

Comment:
Can we add more time information here to make sure this gets preserved?

Reply:
done